### PR TITLE
Create ,ultiple files in modules and add import and exports to differ…

### PR DESCRIPTION
**Pull request by richardoppiyo**
In this milestone:
- Initialized the project with npm
- I created modules folder, added multiple js files and used import and exports to link the files.
- implemented  date time using luxon dependency.

### CODE REVIEW

**Comment by richardoppiyo** [comment]
The Linter error is connected to node_modules. I have tried to find a way to disable that but I can't find it. I am not  sure if its an issue with the version of npm or what? Kindly advice.

**Comment by brytebee** [comment]
Hi, @richardoppiyo 
Have you tried removing and re-installing the linter files?

**Comment by richardoppiyo** [comment]
> Hi, @richardoppiyo Have you tried removing and re-installing the linter files?

Yes, I have tried that but it didn't solve that. I have also re- installed node_modules but it didn't fix the issue

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @richardoppiyo 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request



You have submitted a project with linter errors. Kindly fix the linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Meltrust — COMMENTED** [review]
Hi @richardoppiyo,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not contain a readme file.  Please, remember that for all projects it is mandatory to use the complete [Microverse template](https://github.com/microverseinc/readme-template).   It must contain all sections, except for the "live demo" and "tests" if not required.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: modules/Book.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line import/no-cycle
+
+import Navigator from './navigation.js';
+
+export class Book {
```
Comment (FrederickMih) [inline modules/Book.js:5]: - [ ] Please let's refactor our classes to ES6 functions, this way, we meet the project requirements and get closer to approval.

File: modules/BookList.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+// eslint-disable-next-line import/no-cycle
+import { Book, nav1 } from './Book.js';
+import { DateTime } from '../node_modules/luxon/build/es6/luxon.js';
+
+export default class BookList {
```
Comment (FrederickMih) [inline modules/BookList.js:5]: - [ ] Please let's refactor our classes to ES6 functions, this way, we meet the project requirements and get closer to approval. 
You may consider doing something like `export const bookList = () => {
  code
}`;

File: modules/navigation.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+export default class Navigator {
```
Comment (FrederickMih) [inline modules/navigation.js:1]: - [ ] As mentioned above, please let's refactor our class to ES6 function.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
### Hi @richardoppiyo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by richardoppiyo** [comment]
Re-submitting as advised by CR

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @richardoppiyo 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**